File: OCH_Tao/ABC/A/334.js
const input = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(x => parseInt(x, 10));
if (input[0] > input[1]) {
  console.log("Bat");
} else {
  console.log("Glove");
}

File: OCH_Tao/ABC/B/334.py
#TLEで未AC
A,M,L,R = map(int,input().split())
n = []

for i in range(A,R,M):
  n.append(i)

for i in range(A,L,-M):
  n.insert(0,i)

if L == R:
  print(0)
else:
  print(len(n))